src/types.go

```diff
@@ -34,22 +34,26 @@ type MatchWithKey struct {
 
 // UserProfile wrapper for datastore
 type UserProfile struct {
-	Tournament string
-	Name       string
-	Rating     float64
-	Wins       int
-	Losses     int
-	JoinDate   time.Time
+	Tournament     string
+	Name           string
+	Rating         float64
+        TrueSkillMu    float64
+        TrueSkillSigma float64
+	Wins           int
+	Losses         int
+	JoinDate       time.Time
```

Comment:
You can use a new struct to hold new data.
We don't want to break current table.